backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -39,6 +41,17 @@ const groups = pgTable('groups', {
   updatedAt: timestamp('updated_at').notNull().defaultNow(),
 });
 
+const groupsRelations = relations(groups, ({ many }) => ({
+  users: many(users),
+}));
+
+const usersRelations = relations(users, ({ one }) => ({
+  groups: one(groups, {
```

Comment:
It's one group, so let's name it `group`

Reply:
I didn't know it is possible, thought "groups" here stands for database name.